lib/active_model/validations/chef_version_constraint_validator.rb

```diff
@@ -34,8 +34,9 @@ def initialize(options)
       #
       def validate_each(record, attribute, value)
         Chef::VersionConstraint.new(value)
-      rescue Chef::Exceptions::InvalidVersionConstraint, Chef::Exceptions::InvalidCookbookVersion
-        record.errors.add(attribute, options.fetch(:message), value: value)
+      rescue Chef::Exceptions::InvalidVersionConstraint, Chef::Exceptions::InvalidCookbookVersion => e
+        msg = "#{options.fetch(:message)}. #{e.class}: #{e.message}"
```

Comment:
Are there any cases where `options[:message]` can be empty? If so maybe we should pass a 2nd argument to fetch with a generic message so the message doesn't start with a period. If not, then never mind.